garak/generators/base.py

```diff
@@ -222,3 +222,15 @@ def generate(
                 outputs = self._prune_skip_sequences(outputs)
 
         return outputs
+
+    @staticmethod
+    def conversation_to_list(conversation: Conversation) -> list[dict]:
```

Comment:
Should this be a private function?  It should only be overridden for needs inside a generator.
```suggestion
    def _conversation_to_list(conversation: Conversation) -> list[dict]:
```